contrib/Shopify/sarama/option.go

```diff
@@ -7,15 +7,22 @@ package sarama
 
 import (
 	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
 )
 
 type config struct {
-	serviceName   string
-	analyticsRate float64
+	consumerServiceName string
+	producerServiceName string
+	analyticsRate       float64
 }
 
 func defaults(cfg *config) {
-	cfg.serviceName = "kafka"
+	cfg.producerServiceName = "kafka"
+	cfg.consumerServiceName = "kafka"
+	if svc := globalconfig.ServiceName(); svc != "" {
+		cfg.consumerServiceName = svc
```

Comment:
Should we drop a comment why we're changing one but not the other?

Reply:
I'm not so sure. I feel that if we want to document this, it might be better to do it in one place for all the integrations.

 I think the people who would find that info useful won't be (or hopefully won't have to be) digging through our code to find it. 

Maybe a line in the commit message about it would be appropriate for any person in the future wondering what the change was about.